docs/source/kedro-viz_visualisation.md

```diff
@@ -63,6 +61,16 @@ If a visualisation panel opens up and a pipeline is not visible, refresh the vie
 
 To exit the visualisation, close the browser tab. To regain control of the terminal, enter `^+c` on Mac or `Ctrl+c` on Windows or Linux machines.
 
+Kedro-Viz provides stats related to datasets under the metadata panel
+
+![](./images/pipeline_dataset_stats.png)
+
+These stats are generated by hooks. If you have Kedro-Viz installed and execute `kedro run`, the hooks will generate the stats by default. To disable this, you can disable Kedro-Viz hooks in the settings file of your project.
+
+```{note}
```

Comment:
same here with the code block style

<img width="1108" alt="Screenshot 2024-10-14 at 11 31 37" src="https://github.com/user-attachments/assets/648382d5-fb3a-44a7-8848-5021f784227f">